charts/sonarqube/templates/_pod.tpl

```diff
@@ -19,7 +19,7 @@ metadata:
     {{- toYaml . | nindent 4 }}
     {{- end }}
 spec:
-  automountServiceAccountToken: false
+  automountServiceAccountToken: {{ .Values.podSpec.automountServiceAccountToken }}
```

Comment:
Thank you for your contribution!

Can you please replicate your changes to the `sonarqube-dce` chart?